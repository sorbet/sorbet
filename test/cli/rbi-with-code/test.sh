#!/bin/sh
main/sorbet --silence-dev-message test/cli/rbi-with-code/rbi-with-code.rbi 2>&1
