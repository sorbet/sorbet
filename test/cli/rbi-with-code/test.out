test/cli/rbi-with-code/rbi-with-code.rbi:2: RBI methods must not have code https://srb.help/3010
     2 |  1
          ^
  Autocorrect: Use `-a` to autocorrect
    test/cli/rbi-with-code/rbi-with-code.rbi:2: Delete
     2 |  1
          ^

test/cli/rbi-with-code/rbi-with-code.rbi:6: RBI methods must not have code https://srb.help/3010
     6 |  1
          ^
  Autocorrect: Use `-a` to autocorrect
    test/cli/rbi-with-code/rbi-with-code.rbi:6: Delete
     6 |  1
     7 |  2

test/cli/rbi-with-code/rbi-with-code.rbi:7: RBI methods must not have code https://srb.help/3010
     7 |  2
          ^
  Autocorrect: Use `-a` to autocorrect
    test/cli/rbi-with-code/rbi-with-code.rbi:6: Delete
     6 |  1
     7 |  2
Errors: 3
